vendir.yml

```diff
@@ -7,7 +7,6 @@ directories:
 - path: manifests/cf-deployment
   contents:
   - path: .
-    githubRelease:
-      slug: cloudfoundry/cf-deployment
-      tag: v53.8.0
-      disableAutoChecksumValidation: true
+    git:
+      url: https://github.com/cloudfoundry/cf-deployment
+      ref: v53.8.0
```

Comment:
Ideally the tag would not be hard coded and instead a pattern used so that later ci can auto bump